std/file.d

```diff
@@ -4308,7 +4307,17 @@ string tempDir() @trusted
         }
         else static assert(false, "Unsupported platform");
 
-        if (cache is null) cache = getcwd();
+        if (cache is null)
+        {
+            import std.path : dirSeparator;
+            cache = getcwd() ~ dirSeparator;
+        }
     }
     return cache;
 }
+
+@trusted unittest
+{
+    import std.path : dirSeparator;
+    assert(tempDir[$-1] == dirSeparator);
```

Comment:
This doesn't compile. You need to use `endsWith` or such.

Running the test suite yourself can be complicated but it would be good to at least check if the code compiles before pushing, to avoid wasting the autotesters' resources.